test/handler.js

```diff
@@ -46,15 +46,26 @@ module.exports.exception = api.wrap('GET exception', [], { limit: null }, () =>
   throw Error('Some Exception');
 });
 
+module.exports.deprecation = api.wrap('GET deprecation', [], {
+  deprecated: '1.0.1'
+}, () => api.ApiResponse(''));
+
 module.exports.text = api
   .wrap('GET text', [], () => api.ApiResponse('some text', 200, { 'some-header': 123 }));
 
-module.exports.json = api
-  .wrap('GET json', [], { limit: process.env.RATE_LIMIT }, () => api.JsonResponse({ some: 'json' }));
+module.exports.json = api.wrap('GET json', [],
+  process.env.RATE_LIMIT === undefined
+    ? {}
+    : { limit: parseInt(process.env.RATE_LIMIT, 10) },
+  () => api.JsonResponse({ some: 'json' }));
```

Comment:
just declare limit as a variable at the top of the file... and then declare as undefined or as integer and pass that in here. Much cleaner